File: dayFour/javaScript/taskSix.js
let total = 0;
let evenCount = 0;
for(let count = 1; count <= 10; count++){
    
    const scanner = require("prompt-sync")();
    let number = Number(scanner ("Enter score: "));
    if(number % 2 === 0){
        total += number;
        evenCount += 1;
    }
    
}
let average = total / evenCount;
console.log("The average of the even scores is "+average);

File: dayFour/java/afterBreak/TaskSix.java
import java.util.Scanner;

public class TaskSix{

    public static void main(String[] args){
    
        Scanner input = new Scanner(System.in);
        double total = 0.0;
        int evenCount = 0;
        
        for(int count = 1; count <= 10; count++){
        
            System.out.println("Enter a score: ");
            double number = input.nextInt();
            if(number % 2 == 0){
                total += number;
                evenCount++;
            }
        }
        double average = total / evenCount;
        System.out.println("The average of the even scores is "+average);
    }
}
